content/docs/reference/config/installation-options/index_version_diff.md

```diff
@@ -0,0 +1,55 @@
+| `n/a -----> global.proxy.envoyStatsd.port` | `n/a -----> # example: 9125` | `` |
+
+## `gateways` options
+
+| Key (Old Key ----> New Key) | Default Value (Old Value ----> New Value) | Description (Old Description ---> New Description)|
+| --- | --- | --- |
+| `gateways.istio-ingressgateway.ports.targetPort` | `15443 -----> 80` | `` |
+| `gateways.istio-ingressgateway.ports.name` | `tls -----> http2` | `` |
+| `gateways.istio-ingressgateway.ports.nodePort` | `31400 -----> 31380` | `` |
+| `gateways.istio-ingressgateway.ports.name` | `tls -----> https` | `` |
+| `gateways.istio-ingressgateway.ports.nodePort` | `31400 -----> 31390` | `` |
+| `gateways.istio-ingressgateway.ports.name` | `tls -----> tcp` | `` |
+| `gateways.istio-ingressgateway.ports.targetPort` | `15443 -----> 15029` | `` |
+| `gateways.istio-ingressgateway.ports.name` | `tls -----> https-kiali` | `` |
+| `gateways.istio-ingressgateway.ports.targetPort` | `15443 -----> 15030` | `` |
+| `gateways.istio-ingressgateway.ports.name` | `tls -----> https-prometheus` | `` |
+| `gateways.istio-ingressgateway.ports.targetPort` | `15443 -----> 15031` | `` |
+| `gateways.istio-ingressgateway.ports.name` | `tls -----> https-grafana` | `` |
+| `gateways.istio-ingressgateway.ports.targetPort` | `15443 -----> 15032` | `` |
+| `gateways.istio-ingressgateway.ports.name` | `tls -----> https-tracing` | `` |
+| `gateways.istio-ingressgateway.meshExpansionPorts.targetPort` | `853 -----> 15011` | `` |
+| `gateways.istio-ingressgateway.meshExpansionPorts.name` | `tcp-dns-tls -----> tcp-pilot-grpc-tls` | `` |
+| `gateways.istio-ingressgateway.meshExpansionPorts.targetPort` | `853 -----> 8060` | `` |
+| `gateways.istio-ingressgateway.meshExpansionPorts.name` | `tcp-dns-tls -----> tcp-citadel-grpc-tls` | `` |
+| `gateways.istio-ingressgateway.secretVolumes.secretName` | `istio-ingressgateway-ca-certs -----> istio-ingressgateway-certs` | `` |
+| `gateways.istio-ingressgateway.secretVolumes.mountPath` | `/etc/istio/ingressgateway-ca-certs -----> /etc/istio/ingressgateway-certs` | `` |
+| `gateways.istio-egressgateway.ports.name` | `tls -----> http2` | `` |
+| `gateways.istio-egressgateway.ports.name` | `tls -----> https` | `` |
+| `gateways.istio-egressgateway.secretVolumes.secretName` | `istio-egressgateway-ca-certs -----> istio-egressgateway-certs` | `` |
+| `gateways.istio-egressgateway.secretVolumes.mountPath` | `/etc/istio/egressgateway-ca-certs -----> /etc/istio/egressgateway-certs` | `` |
+| `gateways.istio-ilbgateway.ports.name` | `tcp-dns -----> grpc-pilot-mtls` | `` |
+| `gateways.istio-ilbgateway.ports.name` | `tcp-dns -----> grpc-pilot` | `` |
+| `gateways.istio-ilbgateway.ports.name` | `tcp-dns -----> tcp-citadel-grpc-tls` | `` |
+| `gateways.istio-ilbgateway.secretVolumes.secretName` | `istio-ilbgateway-ca-certs -----> istio-ilbgateway-certs` | `` |
+| `gateways.istio-ilbgateway.secretVolumes.mountPath` | `/etc/istio/ilbgateway-ca-certs -----> /etc/istio/ilbgateway-certs` | `` |
+
```

Comment:
wow, i'm totally surprised on this many changes we introduced in 1.1, thank you @johnma14 for capturing these.

cc @costinm we can use this for the API review